MParT/MultivariateExpansionWorker.h

```diff
@@ -454,6 +416,136 @@ class MultivariateExpansionWorker
 
 private:
 
+    KOKKOS_FUNCTION double GetTermVal(unsigned int termInd, const double* polyCache) const {
```

Comment:
Does this function assume an implicit ordering or the multiindex?  What happens if `nzDims(i)==dim_-1` before we get to the last nonzero entry `end_idx`?  It might be more robust to keep two "termVals": one for the component that will be rectified and one for the others.